src/main/java/teamrtg/rtg/core/RTG.java

```diff
@@ -52,7 +53,19 @@ public void preInit(FMLPreInitializationEvent event) {
         Logger.info("[FMLPreInitializationEvent] Creating RTG's EventManager");
         eventMgr = new EventManagerRTG();
 
-        worldtype = new WorldTypeRTG(ModInfo.MOD_ID);
+        worldtype = new WorldTypeRTG();
+    }
+
+    private void registerStructures() {
+        // Scattered features
+        MapGenStructureIO.registerStructure(MapGenScatteredFeatureRTG.Start.class, "rtg_MapGenScatteredFeatureRTG");
+
+        // Villages
+        if (Mods.RTG.config.ENABLE_VILLAGE_MODIFICATIONS.get())
+            MapGenStructureIO.registerStructure(MapGenVillageRTG.Start.class, "rtg_MapGenVillageRTG");
+
+        // Ocean monuments
+        MapGenStructureIO.registerStructure(StructureOceanMonumentRTG.StartMonument.class, "rtg_MapGenOceanMonumentRTG");
     }
```

Comment:
This was an unnecessary code restructuring. This method was fine where it was. In fact, if you were going to move this method here then you may as well have just remove the method entirely and put those 3 registration lines back into `preInit()`